components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -154,6 +156,7 @@ const BillingSubscriptions = () => {
                     {showCreditCardBanner &&
                         isCardExpired &&
                         creditCardExpiredBanner(setShowCreditCardBanner)}
+                    {(isCloudFreeEnabled && isFreeTrial) && (<CloudTrialBanner trialEndDate={trialEndDate}/>)}
```

Comment:
Should this check also for cloud license?

Reply:
no need, this is in subscription page that is only shown when in cloud.